apps/web-app/src/pages/proofs.tsx

```diff
@@ -52,12 +52,10 @@ export default function ProofsPage() {
             setLogs(`Posting your anonymous feedback...`)
 
             try {
-                const group = new Group(env.GROUP_ID)
+                const group = new Group(env.GROUP_ID, undefined, _users)
```

Comment:
We could even explicit the tree depth parameter here. What do you think?

Reply:
Yes, I agree, it would be easier to understand the parameters of the Group class.